Classes/Service/JsonServer.php

```diff
@@ -118,7 +118,7 @@ protected function getServiceUri()
     {
         $uri = GeneralUtility::getIndpEnv('TYPO3_REQUEST_URL');
         $parts = parse_url($uri);
-        $parts['query'] = 'eID=JsonServer&amp;server=' . $this->serverKey;
+        $parts['query'] = 'eID=JsonServer&server=' . $this->serverKey;
```

Comment:
Add changed it in the SoapServer, but in the JsonServer it is wrong.

So SoapServer needs encoded Ampersand, but JsonServer does not.